feathr-example/scripts/update_ip_address.sh

```diff
@@ -0,0 +1,44 @@
+#!/bin/bash          
```

Comment:
Where is this script used? i.e. when should the user run this?

Reply:
Script is no longer required. 
Based on offline chat with @daxsorbito : this script was used as a workaround for a bug related to spark configuration in the source feathr repo which needed the ip address to be refreshed. The bug was fixed in the source repo; So this script can be removed.